x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-studio.adoc

```diff
@@ -0,0 +1,293 @@
+= Use Anypoint Studio to Configure X12 EDI Connector - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure an input source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<conigure-other-fields,Configure the other connector fields>>.
+
+When you run the connector, you can view the app log to check for problems, as described in <<view-app-log,View the App Log>>.
+
+You must change the default Anypoint configuration to use the X12 EDI connector 2.x.
+
+NOTE: If Anypoint Studio 7.1.0 and later requires more than the standard memory
+configuration to use the X12 EDI connector 2.x, then you'll need to change the memory
+setting for Studio. Edit the `Anypoint.ini` file in the Anypoint Studio
+installation directory and change the two lines starting with `-Xms` and `-Xmx` to:
+
+[source,text,linenums]
+----
+-Xms4096m
+-Xmx4096m
+----
+
+Setting the Anypoint Studio memory to 4 GB, which is sufficient
+for the most common uses of this connector.
+////
+// If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:x12-edi-connector-reference.adoc[X12 EDI Reference].
+////
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add X12 EDI (X12 EDI Connector) to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type `X12 EDI:Configuration` in the search field.
+. Click X12 EDI:Configuration in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with X12 EDI Connector:
```

Comment:
The template has two default sources (Scheduler and HTTP). If you are going to remove the Scheduler one, then change the above line so it's not saying "one of these sources"